packages/manager/src/features/CloudPulse/Alerts/ContextualView/AlertInformationActionTable.tsx

```diff
@@ -151,10 +155,8 @@ export const AlertInformationActionTable = (
     resetToInitialState,
   } = useContextualAlertsState(alerts, entityId);
 
-  const { mutateAsync: updateAlerts } = useServiceAlertsMutation(
-    serviceType,
-    entityId ?? ''
-  );
+  // Mutation to update alerts as per service type
+  const updateAlerts = useAlertsMutation(serviceType, entityId ?? '');
```

Comment:
```suggestion
  const updateAlerts = useAlertsMutation(serviceType, entityId);
```

lets not use any fallback value instead handle undefined separately